glances_api/__init__.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         ssl=False,
         username=None,
         password=None,
+        custom_httpx_client=None,
```

Comment:
I'd just call this `httpx_client`.
```suggestion
        httpx_client=None,
```